main.tf

```diff
@@ -1,16 +1,16 @@
 module "landing_zone" {
-  source                  = "MitocGroup/landing-zone/aws"
-  terraform_config        = false
+  source                  = "./modules/landing_zone_reader"

```

Comment:
Here you must put the path to LZ.
`source                  = "MitocGroup/landing-zone/aws"`